koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -234,14 +244,21 @@ def process(self, parquet_base_filename, daily_data_frames):
                 cluster_topology = accessor.get_openshift_topology_for_multiple_providers(ocp_provider_uuids)
             # Get matching tags
             matched_tags = self.get_matched_tags(ocp_provider_uuids)
-            for i, daily_data_frame in enumerate(daily_data_frames):
-                openshift_filtered_data_frame = self.ocp_on_cloud_data_processor(
-                    daily_data_frame, cluster_topology, matched_tags
-                )
+            daily_data_frames = pd.concat(daily_data_frames, ignore_index=True)
+            openshift_filtered_data_frame = self.ocp_on_cloud_data_processor(
+                daily_data_frames, cluster_topology, matched_tags
+            )
 
-                if self.provider_type in (Provider.PROVIDER_GCP, Provider.PROVIDER_GCP_LOCAL):
-                    self.create_partitioned_ocp_on_cloud_parquet(
-                        openshift_filtered_data_frame, parquet_base_filename, i
-                    )
-                else:
-                    self.create_ocp_on_cloud_parquet(openshift_filtered_data_frame, parquet_base_filename, i)
+            if self.provider_type in (
+                Provider.PROVIDER_GCP,
+                Provider.PROVIDER_GCP_LOCAL,
+                Provider.PROVIDER_AWS,
+                Provider.PROVIDER_AWS_LOCAL,
+                Provider.PROVIDER_AZURE,
+                Provider.PROVIDER_AZURE_LOCAL,
+            ):
+                self.create_partitioned_ocp_on_cloud_parquet(
+                    openshift_filtered_data_frame, parquet_base_filename, manifest_id
+                )
+            else:
+                self.create_ocp_on_cloud_parquet(openshift_filtered_data_frame, parquet_base_filename, file_number=0)
```

Comment:
Do we need this else if we are calling the previous method for AWS, Azure, and GCP.

Reply:
Probably not, but I didnt want to remove all the code for all of these yet. Id rather fix and clean up later.